users/views.py

```diff
@@ -92,15 +96,8 @@ def get(self, request: 'HttpRequest', *args, **kwargs) -> HttpResponse:
         :return: A response with the rendered
                  :obj:`template <EditUser.template_name>`.
         """
-        form = UserProfileForm(
-            user_id=self.request.user.id,
-            email=self.profile.user.email,
-            username=self.profile.user.username,
-            first_name=self.profile.user.first_name,
-            last_name=self.profile.user.last_name,
-            avatar=self.profile.avatar,
-            bio=self.profile.bio
-        )
+        form = UserProfileForm(instance=self.profile,
+                               initial={'user_id': self.request.user.id})
```

Comment:
`initial` is not needed since you removed the `user_id` field.

Reply:
That's true, I refactored that piece from initial to instance to removing user_id entirely (since auth is done via the request anyway) and forgot to remove that.